### ISSUE TITLE
Python 3 compatability

### ISSUE THREAD

**Issue by esc** [seveas/git-spindle#113]
I discovered today that `git-spindle` doesn't seem to support Python 3.

```
esc@omnitech ~/tbn/20bn-api (git)-[fix_json_schema_tag_image] % git hub pull-request
Traceback (most recent call last):
  File "/home/esc/bin/git-hub", line 9, in <module>
    from gitspindle.github import GitHub
  File "/home/esc/.venv/lib/python3.5/site-packages/gitspindle/__init__.py", line 394
    print repo
             ^
SyntaxError: Missing parentheses in call to 'print'
1 esc@omnitech ~/tbn/20bn-api (git)-[fix_json_schema_tag_image] % python --version                                                                                    :(
Python 3.5.1+
```

**Comment by seveas** [comment]
git-spindle actually does support python 3, and this bug has been fixed.

**Comment by esc** [comment]
Cool, I'll start using it again. \o/

On 19 September 2017 20:54:12 CEST, Dennis Kaarsemaker <notifications@github.com> wrote:
>Closed #113.
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/seveas/git-spindle/issues/113#event-1256073622

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.
